compiler/mio-circle08/src/Reader.cpp

```diff
@@ -118,7 +120,10 @@ size_t Reader::buffer_info(uint32_t buf_idx, const uint8_t **buff_data, bool &ex
         return 0;
 
       ext_offset = true;
-      *buff_data = reinterpret_cast<const uint8_t *>(&_rawdata->at(buffer_offset));
+      if (buff_data != nullptr)
```

Comment:
some modules only want size and calls with null `buff_data`.

Reply:
Then, if `buff_data` is `nullptr`, `buffer->size()` is zero?

Reply:
> Then, if buff_data is nullptr, buffer->size() is zero?

`buff_data` is return args, and if null, call doesn't want the buff_data ptr.
`buffer->size()` can have non zero value as it's size of `buffer`

this code change is same as below code for `else if (auto *array = buffer->data())` case.